tests/tests/control_debug

```diff
@@ -25,6 +25,7 @@ export OUTPUT_GRID='gaussian_grid'
 export NSTF_NAME='2,0,0,0,0'
 export IAER=5111
 export FHMAX=1
+export WRITE_DOPOST=.true.
```

Comment:
If this tests now runs inline post, shouldn't we compare post grib files against the baselines?